### ISSUE TITLE
[Rule Suggestion] ordered-imports: Allow for optional new lines between groups.

### ISSUE THREAD

**Issue by omiiii** [palantir/tslint#3450]
This is more of a preference of wanting to only have new line separation between libraries and directory imports while maintaining the sorted order enforced with "grouped-imports".

Here's an example of what I mean:
#### Enforces this
```ts
import * as fs from "fs";

import { Foo } from "../no";

import { Foo } from "./no";
```
#### and can optionally enforce this
```ts
import * as fs from "fs";

import { Foo } from "../no";
import { Foo } from "./no";
```
Essentially, an option to have an enforced order of imports, but not the new lines.

**Comment by adidahiya** [comment]
To clarify, you want the `"grouped-imports"` option but more granular control over what the groups are? I think that makes sense; we'd accept a new rule option that just separates external vs. internal paths.

**Comment by KrzysztofJaneczko** [comment]
Separating groups of imports is something I'd really like to have, too. Currently I use eslint with eslint-plugin-import to manage import groups and this would allow me to drop eslint entirely.

**Comment by omiiii** [comment]
@adidahiya Wow, I couldn't be worse with timing here. Sorry!
To answer your question, that's exactly what I'm proposing. Having groups is great, but being able to designate what the groups are would be even better!

**Comment by bschlenk** [comment]
This should also allow you to define the criteria for a group as well. I have my tsconfig `paths` set up so that I can absolute import from src by prefixing with `@/`. Then I group like this:

```
import * as fs from "fs";

import { Foo } from "@/components/Foo";

import { Bar } from "./Bar";
```

**Comment by abierbaum** [comment]
@omiiii Does  #4134 give you what you need/want?

**Comment by omiiii** [comment]
@abierbaum Hey! Yes, that's exactly the kind of customization I was looking for. Thanks!

**Comment by adidahiya** [comment]
I believe this was fixed by #4134, please file a new issue if needed
